src/test/java/seedu/linkedout/model/applicant/NameContainsKeywordsPredicateTest.java

```diff
@@ -75,6 +75,6 @@ public void test_nameDoesNotContainKeywords_returnsFalse() {
         // Keywords match phone, email and linkedout, but does not match name
         predicate = new NameContainsKeywordsPredicate(Arrays.asList("12345", "alice@email.com", "Main", "Street"));
         assertFalse(predicate.test(new ApplicantBuilder().withName("Alice").withPhone("12345")
-                .withEmail("alice@email.com").withAddress("Main Street").build()));
+                .withEmail("alice@email.com").withJob("Main Street").withStage("Main Street").build()));
```

Comment:
Change "Main Street" to a job

Reply:
90a61b304fe21ba8d21974d7756c14d2c13b5405